client/cody-ui/src/chat/BlinkingCursor.tsx

```diff
@@ -2,4 +2,9 @@ import React from 'react'
 
 import styles from './BlinkingCursor.module.css'
 
-export const BlinkingCursor: React.FunctionComponent = () => <span className={styles.cursor} />
+export const BlinkingCursor: React.FunctionComponent = () => (
```

Comment:
```suggestion
export const BlinkingDots: React.FunctionComponent = () => <span className={styles.blinkDots} />
```
Since this is a reusable component I think it would be better if we could rename it accordingly (from `BlinkingCursor` to `BlinkingDot` including file names etc) and replace the old one in this [file line 96](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/client/cody-ui/src/chat/TranscriptItem.tsx?L96), where we just added a new message `Fetching context...` in the new release so you can replace it with the `working on it` message, what do you think 😄 ?

Other than that I think it looks great! Thank you so much!

Reply:
This sounds good to me. It would definitely be better to rename the component to BlinkingDot. Will make this change and would also update the old one you mentioned. 
Thanks a lot for the review :)

Reply:
Hey @deepak2431 hope it's not too late but there is an ongoing discussion about keeping the blinkingCursor instead of replacing it with text and blinkingDots, so I think it might be better to leave the blinkingCusor on for now, apologies for the trouble caused 🙇‍♀️

Reply:
No problem @abeatrix , happens sometimes as part of development :)
Shall I discard the changes of BlinkingDots and update this PR? As apart from that this PR has two other small changes.